chainregistry.go

```diff
@@ -262,6 +263,22 @@ func newChainControlFromConfig(cfg *config, chanDB *channeldb.DB,
 			activeNetParams.Params, neutrinoCS,
 		)
 
+		// Fix a timing issue with Neutrino in our itests. Because the
+		// unlock of the wallet is now much faster with scrypt tuned
+		// down, Neutrino doesn't have enough time to sync block headers
+		// to know how many blocks there are in total before we start
+		// with the recovery. This can lead to the rescan missing some
+		// of the addresses. By just waiting half a second we can fix
+		// this flake in our integration tests. This cannot happen on
+		// testnet or mainnet because there we always wait for the
+		// wallet to be fully synced before we even start the recovery.
+		network := activeNetParams.Params.Net
+		if network == chaincfg.RegressionNetParams.Net ||
+			network == chaincfg.SimNetParams.Net {
+
+			time.Sleep(500 * time.Millisecond)
```

Comment:
I'd prefer not having this sleep in production code, I wonder if we could have this be somewhere in the integration tests instead? Might not be possible though given that there isn't a hook between initializing the neutrino node and the wallet.

Reply:
Yeah, I don't really like that either. I tried adding it only to the itest but as you say, there is no hook for it. That's why I created https://github.com/btcsuite/btcwallet/pull/681 in the first place, to fix the problem instead of a workaround. Now that it got merged, I think it's better to use the fix instead of the wait. Switched commits.